NoClassDefFoundError when running TCK tests against the master branch...

The `tck/pom.xml` file in the `master` branch specifies an exclusion on the `io.rest-assured:rest-assured` dependency, as follows:

```
         <dependency>
             <groupId>io.rest-assured</groupId>
             <artifactId>rest-assured</artifactId>
             <version>${rest-assured.version}</version>
             <exclusions>
               <exclusion>
                 <groupId>io.rest-assured</groupId>
                 <artifactId>xml-path</artifactId>
               </exclusion>
             </exclusions>
         </dependency>
```
This results in all of the `master` branch TCK tests failing on the client side with the following exception:

```
java.lang.NoClassDefFoundError: io.restassured.mapper.factory.JAXBObjectMapperFactory 
at io.restassured.config.RestAssuredConfig.<init>(RestAssuredConfig.java:41) 
at io.restassured.RestAssured.<clinit>(RestAssured.java:420) 
at org.eclipse.microprofile.openapi.tck.AppTestBase.configureRestAssured(AppTestBase.java:65) 
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) 
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) 
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) 
at java.base/java.lang.reflect.Method.invoke(Method.java:566) 
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85) 
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510) 
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211) 
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138) 
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170) 
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104) 
at org.testng.TestRunner.privateRun(TestRunner.java:774) 
at org.testng.TestRunner.run(TestRunner.java:624) 
at org.testng.SuiteRunner.runTest(SuiteRunner.java:359) 
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354) 
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312) 
at org.testng.SuiteRunner.run(SuiteRunner.java:261) 
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) 
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86) 
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215) 
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140) 
at org.testng.TestNG.run(TestNG.java:1048) 
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115) 
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204) 
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107) 
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112) 
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200) 
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153) 
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103) 
Caused by: java.lang.ClassNotFoundException: io.restassured.mapper.factory.JAXBObjectMapperFactory 
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:766) 
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178) 
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:1077) 
... 31 more
```
This is due to the following dependency chain in the rest-assured code:
```
    io.restassured.config.RestAssuredConfig
        |
        ---> io.restassured.config.XmlConfig
            |
            ----> io.restassured.path.xml.XmlPath
                |
                ----> io.restassured.mapper.factory.JAXBObjectMapperFactory
```
In order to avoid this exceptin, the exclusion on the `xml-path` component of rest-assured should be removed.

@MikeEdgar Using `4.3.0` with the `xml-path` exclusion results in the same error... although the stacktrace is slightly different:
```
java.lang.NoClassDefFoundError: io.restassured.RestAssured (initialization failure) 
at java.base/java.lang.J9VMInternals.initializationAlreadyFailed(J9VMInternals.java:143) 
at org.eclipse.microprofile.openapi.tck.AppTestBase.configureRestAssured(AppTestBase.java:65) 
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) 
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) 
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) 
at java.base/java.lang.reflect.Method.invoke(Method.java:566) 
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124) 
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59) 
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458) 
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222) 
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142) 
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168) 
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105) 
at org.testng.TestRunner.privateRun(TestRunner.java:648) 
at org.testng.TestRunner.run(TestRunner.java:505) 
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455) 
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450) 
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415) 
at org.testng.SuiteRunner.run(SuiteRunner.java:364) 
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) 
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84) 
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208) 
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137) 
at org.testng.TestNG.runSuites(TestNG.java:1049) 
at org.testng.TestNG.run(TestNG.java:1017) 
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132) 
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) 
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) 
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147) 
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290) 
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242) 
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121) 
Caused by: java.lang.NoClassDefFoundError: io.restassured.path.xml.mapper.factory.JAXBObjectMapperFactory 
at io.restassured.config.RestAssuredConfig.<init>(RestAssuredConfig.java:41) 
at io.restassured.RestAssured.<clinit>(RestAssured.java:421) 
... 31 more 
Caused by: java.lang.ClassNotFoundException: io.restassured.path.xml.mapper.factory.JAXBObjectMapperFactory 
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:766) 
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178) 
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:1077) 
... 33 more
``` 
So it looks like we need to remove the exclusion regardless of which version of `rest-assured` we use.

--- a/pom.xml
+++ b/pom.xml
@@ -43,7 +43,7 @@
         <arquillian.version>1.1.13.Final</arquillian.version>
         <testng.version>7.0.0</testng.version>
         <jaxrs.version>2.1.6</jaxrs.version>
-        <rest-assured.version>3.0.6</rest-assured.version>
+        <rest-assured.version>4.3.0</rest-assured.version>
         <hamcrest-all.version>1.3</hamcrest-all.version>
         <java-hamcrest.version>2.0.0.0</java-hamcrest.version>
         <httpclient.version>4.5.2</httpclient.version>

--- a/tck/pom.xml
+++ b/tck/pom.xml
@@ -128,12 +128,6 @@
              <groupId>io.rest-assured</groupId>
              <artifactId>rest-assured</artifactId>
              <version>${rest-assured.version}</version>
-             <exclusions>
-               <exclusion>
-                 <groupId>io.rest-assured</groupId>
-                 <artifactId>xml-path</artifactId>
-               </exclusion>
-             </exclusions>
          </dependency>
 
          <dependency>

